aqi_monitor/apps/monitor/function_for_task.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime as DT, timedelta
+
+def out_message(aqi):
```

Comment:
1) Let's add types annotation:
https://docs.python.org/3/library/typing.html

2) I'm not sure that function's name represents it's main purpose. Please consider another name

3) Let's add some comments 🙂
- https://www.python.org/dev/peps/pep-0257/
- https://realpython.com/documenting-python-code/

Reply:
Fixed